src/internal/m365/onedrive/mock/handlers.go

```diff
@@ -47,7 +47,7 @@ type BackupHandler struct {
 
 func DefaultOneDriveBH() *BackupHandler {
 	return &BackupHandler{
-		ItemInfo:     details.ItemInfo{OneDrive: &details.OneDriveInfo{}},
+		ItemInfo:     details.ItemInfo{OneDrive: &details.OneDriveInfo{}, Extension: &details.ExtensionInfo{}},
```

Comment:
It may be a good time to introduce an ItemInfo factory, to avoid gotchas from forgetting to initialize this or that property.

Reply:
Sounds good. Let me add this in a later PR.